src/core/a-assets.js

```diff
@@ -83,9 +83,11 @@ registerElement('a-asset-item', {
     attachedCallback: {
       value: function () {
         var self = this;
-        var src = this.getAttribute('src');
+        var src;
 
+        src = this.getAttribute('src');
```

Comment:
I updated it. I guess I will just use discretion on when to declare + assign (small functions), and when to declare + assign later (large functions or functions with multiple paths).